<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>R9PcbTesterWidget</class>
 <widget class="QWidget" name="R9PcbTesterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>680</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Noto Sans KR</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="pcbTestResultLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>240</width>
     <height>240</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="pcbTestProgressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pcbTestButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>1181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TEST</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>931</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayoutPcbConsole"/>
  </widget>
  <widget class="QListView" name="pcbTestHistoryListView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>721</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="pcbTestStatsWidget" native="true">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>420</y>
     <width>451</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
